### PR TITLE
Add validation video and collisions example to docs and blog post

### PR DESCRIPTION
**Pull request by ivarpruijn**
Adds the demo video about the new validation features to:
- The docs, under core-concepts > validation
- The blog post about the release, replacing the social image after the intro.

Also adds an example about collisions to the validation docs.

### CODE REVIEW

File: src/components/vimeo-player/index.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ import {
 function VimeoPlayer({ 
   videoId, 
   id, // required if the Video needs to be tracked
+  paddingBottom = "58.25%", // the ratio  for the video, e.g. 58.25% for 16:9 videos
```
Comment (ivarpruijn) [inline src/components/vimeo-player/index.js:17]: Added to deal with different aspect ratios of videos, like the 2-minute intro vs these demo videos.

**Review by jansenbob — APPROVED** [review]
LGTM. I think we could take some steps in the recording of videos to make them a bit more suited for smaller screens though. I don't think you would be able to follow this with a small, sub par resolution screen.

**Comment by sdirosa** [comment]
> LGTM. I think we could take some steps in the recording of videos to make them a bit more suited for smaller screens though. I don't think you would be able to follow this with a small, sub par resolution screen.

Another solution is to make the demo application used in the video public, so people can play around with it.

Or make a CodeSandbox / StackBlitz

**Comment by ivarpruijn** [comment]
> 

That's 

> > LGTM. I think we could take some steps in the recording of videos to make them a bit more suited for smaller screens though. I don't think you would be able to follow this with a small, sub par resolution screen.
> 
> Another solution is to make the demo application used in the video public, so people can play around with it.
> 
> Or make a CodeSandbox / StackBlitz

Also a good idea, let's keep it in mind. Merging this for now.

File: src/components/vimeo-player/index.js (hunk 1 above)
Comment (ivarpruijn) [reply src/components/vimeo-player/index.js:17]: Good idea. Will make a note to see if that works later.

**Pull request merged**